interface/HH_BTag.h

```diff
@@ -13,24 +13,24 @@ namespace InputVars{
 
 class HH_BTag {
 public:
-    HH_BTag(const std::string& model);
+    HH_BTag(const std::vector<std::string>& models);
```

Comment:
- define `static constexpr size_t n_models = 2;` and use it everywhere instead of 2.
- `std::vector<std::string>` -> `std::array<std::string, n_models>`